cli/src/commands/init.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::ArgMatches;
+use clap::Error;
+use utils;
+
+pub fn run<'a>(args: &ArgMatches<'a>) -> Result<(), Error> {
+    let rest_api_url = args.value_of("url").unwrap();
+    utils::set_config(rest_api_url)?;
+    println!(
```

Comment:
Maybe use the `info!` macro instead of `println`? Then you can use to control the verbosity level, something like 
here https://github.com/hyperledger/sawtooth-sabre/blob/master/tp/src/main.rs#L43